semconvgen/generator.go

```diff
@@ -379,6 +388,30 @@ var capitalizations = []string{
 	"OTel",
 }
 
+func capitalizations(cfg config) ([]string, error) {
```

Comment:
Can we add unit tests?

Reply:
@pellared We can add unit tests, but I'm not sure if that's in scope for this ticket since nothing else is being tested? Would it be better to create an issue "semconvgen: add unit tests"?

Reply:
Sounds good, I added some unit tests for the new functionality, thanks @pellared!

Reply:
Now with this testing in place, should I add the issue "semconvgen: improve unit test coverage"?

Reply:
@pellared checking on this again, I could make this ticket and add the rest of the testing for this file?

Reply:
Sure